### PR TITLE
Add sumologic url to icds localsettings

### PR DESCRIPTION
**Pull request by proteusvacuum**
@biyeun @snopoke

### CODE REVIEW

File: environments/icds-new/public.yml [hunk 1]
```diff
@@ -404,6 +404,7 @@ localsettings:
   SMS_GATEWAY_PARAMS:
   SMS_GATEWAY_URL: 
   SMS_QUEUE_ENABLED: True
+  SUMOLOGIC_URL: "{{ localsettings_private.SUMOLOGIC_URL }}"
```
Comment (snopoke) [inline environments/icds-new/public.yml]: Also needs to be added into the localsettings file.

Comment (proteusvacuum) [reply environments/icds-new/public.yml]: I added it to `settings.py` and the vault file... 

https://github.com/dimagi/commcare-hq/pull/19597/files#diff-bdf3ecebd8379ca98cc89e545fc90899

Where else do I need to add it?

Comment (snopoke) [reply environments/icds-new/public.yml]: In `roles/commcarehq/templates/localsettings.py.yml` (I think that path is right)

Comment (proteusvacuum) [reply environments/icds-new/public.yml]: I see, thanks!

**Comment by emord** [comment]
HQ pr was merged

**Comment by proteusvacuum** [comment]
this will need to be run on icds before the sumologic logs will start getting sent.

**Pull request merged**